src/phoenix/config.py

```diff
@@ -51,12 +51,21 @@
 ENV_PHOENIX_SERVER_INSTRUMENTATION_OTLP_TRACE_COLLECTOR_GRPC_ENDPOINT = (
     "PHOENIX_SERVER_INSTRUMENTATION_OTLP_TRACE_COLLECTOR_GRPC_ENDPOINT"
 )
+ENV_PHOENIX_SERVER_INSTRUMENTATION_OTLP_METRICS_COLLECTOR_HTTP_ENDPOINT = (
+    "PHOENIX_SERVER_INSTRUMENTATION_OTLP_METRIC_COLLECTOR_HTTP_ENDPOINT"
+)
+ENV_PHOENIX_SERVER_INSTRUMENTATION_OTLP_METRICS_COLLECTOR_GRPC_ENDPOINT = (
+    "PHOENIX_SERVER_INSTRUMENTATION_OTLP_METRIC_COLLECTOR_GRPC_ENDPOINT"
+)
 
 
 def server_instrumentation_is_enabled() -> bool:
     return bool(
         os.getenv(ENV_PHOENIX_SERVER_INSTRUMENTATION_OTLP_TRACE_COLLECTOR_HTTP_ENDPOINT)
-    ) or bool(os.getenv(ENV_PHOENIX_SERVER_INSTRUMENTATION_OTLP_TRACE_COLLECTOR_GRPC_ENDPOINT))
+        or os.getenv(ENV_PHOENIX_SERVER_INSTRUMENTATION_OTLP_TRACE_COLLECTOR_GRPC_ENDPOINT)
+        or os.getenv(ENV_PHOENIX_SERVER_INSTRUMENTATION_OTLP_METRICS_COLLECTOR_HTTP_ENDPOINT)
```

Comment:
Isn't it normal to just have a OTLP endpoint and for that to just support metrics too?